CovertInfiltration/Src/CovertInfiltration/Classes/X2StrategyElement_InfiltrationRewards.uc

```diff
@@ -11,6 +11,8 @@ class X2StrategyElement_InfiltrationRewards extends X2StrategyElement config(Gam
 var config array<int> SmallIntelRewardMin;
 var config array<int> SmallIntelRewardMax;
 
+var localized string strDarkEventReward;
```

Comment:
Where is this used?

Reply:
Oh I forgot to delete that after I realized that I could use the reward localization instead